src/hooks/useTodoTask.ts

```diff
@@ -72,6 +72,12 @@ export const useTodoTask = () => {
   const onDeleteTodo = (todoId: number) => {
     deleteTodo(todoId);
   };
+  const onEditTodo = (todoToEdit: ITodo) => {
+    setTodo(todoToEdit);
+  };
+  const onSaveEditTodo = () => {
+    editTodo(todo);
+  };
```

Comment:
Use one style of code - indents between functions

Reply:
Added indents between functions.